index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Chris Closser</title>
+    <!-- <link rel="stylesheet" href="vendor/styles/reset.css"> -->
```

Comment:
Don't forget to comment this back in! It is important. It takes away all built in spacing and sizing to give you a blank slate to manipulate!